librz/io/p/io_srec.c

```diff
@@ -0,0 +1,810 @@
+// SPDX-FileCopyrightText: 2013-2017 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2013-2017 fenugrec <fenugrec@users.sourceforge.net>
+// SPDX-License-Identifier: LGPL-3.0-only
+// Contributed by manouser <manolache.alexandru8@gmail.com>
```

Comment:
if you have implemented this, why you give pancake and fenugrec the Copyright ?

Reply:
I implemented it, yes, but I had the ihex implementation as an example, which was made by those 2, so I figured out I should give them some credit.

Reply:
no need. put yourself.